sources/api/netdog/src/main.rs

```diff
@@ -24,7 +24,7 @@ valid example: `netdog.default-interface=eno1:dhcp4,dhcp6?`.
 The subcommand `write-resolv-conf` writes the resolv.conf, favoring DNS API settings and
 supplementing any missing settings with DNS settings from the primary interface's DHCP lease.  It
 is meant to be used as a restart command for DNS API settings.
-*/
+ */
```

Comment:
Is this happening from `cargo fmt`? If not I wouldn't include the change:
```suggestion
*/
```